src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -3692,6 +3697,1485 @@ explicit_encryption_destroy (ee_fixture *eef)
+                                   BCON_DOUBLE (30),
+                                   "}",
+                                   "}",
+                                   "]");
+      mongoc_cursor_t *cursor =
+         test_explicit_encryption_range_agg_helper (eef_range, eef);
+      ASSERT (mongoc_cursor_next (cursor, &got));
+      ASSERT_MATCH (got, "{ 'encryptedDoublePrecision': 30.0}");
+      ASSERT (!mongoc_cursor_next (cursor, &got) &&
+              "expected one document, got more than one");
+   }
+   // aggregate command to return zero documents
+   // The difference between this and the previous test is using lt and not lte
+   {
+      const bson_t *got;
+      eef_range->query = BCON_NEW ("$and",
+                                   "[",
+                                   "{",
+                                   "encryptedDoublePrecision",
+                                   "{",
+                                   "$gte",
+                                   BCON_DOUBLE (7),
+                                   "}",
+                                   "}",
+                                   "{",
+                                   "encryptedDoublePrecision",
+                                   "{",
+                                   "$lt",
+                                   BCON_DOUBLE (30),
+                                   "}",
+                                   "}",
+                                   "]");
+      mongoc_cursor_t *cursor =
+         test_explicit_encryption_range_agg_helper (eef_range, eef);
+      ASSERT (!mongoc_cursor_next (cursor, &got) &&
+              "expected zero documents, got more than none");
+   }
+   explicit_encryption_range_destroy (eef_range);
+   explicit_encryption_destroy (eef);
+}
+
+static void
+test_explicit_encryption_range_int_error (void *unused)
+{
+   ee_fixture *eef = explicit_encryption_setup (
+      "./src/libmongoc/tests/client_side_encryption_prose/explicit_encryption/"
+      "range-encryptedFields-Int.json");
+   ee_range_test_fixture *eef_range =
+      (ee_range_test_fixture *) bson_malloc0 (sizeof (ee_range_test_fixture));
+   eef_range->field_name = "encryptedInt";
+   eef_range->sparsity = 1;
+   eef_range->minmax.set = true;
+   eef_range->minmax.min.value_type = BSON_TYPE_INT32;
+   eef_range->minmax.min.value.v_int32 = 0;
+   eef_range->minmax.max.value_type = BSON_TYPE_INT32;
+   eef_range->minmax.max.value.v_int32 = 250;
+
+   /* Can't encrypt document that is greater than max */
+   {
+      eef_range->doc_value[0].value_type = BSON_TYPE_INT32;
+      eef_range->doc_value[0].value.v_int32 = 251;
+      bson_error_t error =
+         test_explicit_encryption_range_error_helper (eef_range, eef);
+      ASSERT_ERROR_CONTAINS (
+         error,
+         MONGOC_ERROR_CLIENT_SIDE_ENCRYPTION,
+         MONGOC_ERROR_STREAM_INVALID_TYPE,
+         "Value must be greater than or equal to the minimum value and less "
+         "than or equal to the maximum value");
```

Comment:
```suggestion
         "");
```

Suggest removing the strings from the `ASSERT_ERROR_CONTAINS` statements. If libmongocrypt changes the error messages, it would require drivers relying on those strings to update tests.